code/modules/vending/autodrobe.dm

```diff
@@ -140,7 +140,9 @@
 				   /obj/item/clothing/suit/chaplainsuit/clownpriest = 1,
 				   /obj/item/clothing/head/clownmitre = 1,
 				   /obj/item/skub = 1,
-				   /obj/item/clothing/suit/hooded/mysticrobe = 1)
+				   /obj/item/clothing/suit/hooded/mysticrobe = 1,
+				   /obj/item/clothing/under/dress/wedding_dress = 1,
+				   /obj/item/clothing/under/suit/tuxedo = 1)
```

Comment:
they shouldnt be above skub...

Reply:
undocumented and shouldnt be here, clowns dont need wedding dress

Reply:
it is in fact documented